app/workers/aggregation_data_mailer_worker.rb

```diff
@@ -1,5 +1,4 @@
 class AggregationDataMailerWorker
-  include Sidekiq::Worker
   include DumpWorker
```

Comment:
Yep - i did it this was as the `DumpWorker` is a dumping ground of responsibilities. I'll refactor and cleanup.